### ISSUE TITLE
No auto-scroll on drag

### ISSUE THREAD

**Issue by fritz-c** [fritz-c/react-sortable-tree#3]
Excluding Google Chrome, which scrolls elements when you drag child elements inside them, no browsers move the screen when you drag the element near the outer edge.

**Comment by firatsarlar** [comment]
Please checkout https://github.com/martindrapeau/jQueryDndPageScroll/blob/master/jquery.dnd_page_scroll.js

Also working example
https://jsfiddle.net/0vv7fs63/2/

I'm not suggesting to use jquery plugin but checking source code.
Simply adding two divs ( top and bottom of the page ) and listening drag over events, and scrolling accordingly is the solution.

**Comment by fritz-c** [comment]
Thank you! I hadn't thought of that approach. I'll give it a shot today if I have some time.

**Comment by fritz-c** [comment]
I ended up taking a different approach to make this work (autoscroll is now available in v0.1.2).

I initially tried the method you suggested, but the problem with adding divs is that they would block other mouse operations (click, drag, hover, etc.). The approach I used in the end was just using a ref with the dimensions of the onscreen element, and scrolling when the offset approached those dimensions.

Anyway, thanks a bunch for the advice. It inspired me to start implementing the feature.
